src/components/WishListCard/WishListCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { FaStar,FaRupeeSign,FaHeart} from "react-icons/fa";
+import "../../pages/WishListPage/WishListPage.css";
+import "../ProductCard/Productcard"
+
+
+
+ export const WishListCard = () => {
+  return (
+ 
+     <div className="card-for-ecommerce pd-1 card-with-shadow">
+            <div className="card-image-holder">
+              <img
+                src="./assets/MEN/Acqua-di-Gio-Essenza-Giorgio-Armani_men.jpg"
+                className="card-image"
```

Comment:
can use variable for images from assets.so, if you have to change image in future it will be easy to just change import.
For eg:
`import storeLogo from "../../assets/location.png";
 <img src={storeLogo}  />`

Reply:
will do